.github/workflows/build.yml

```diff
@@ -17,7 +18,6 @@ jobs:
       GEMS_NODE_MODULES_IMAGE: ghcr.io/dfe-digital/apply-teacher-training-gems-node-modules
     outputs:
       IMAGE_TAG: ${{ env.IMAGE_TAG }}
-      GIT_BRANCH: ${{ env.GIT_BRANCH }}
```

Comment:
GIT_BRANCH is required in the jobs below, at least for the slack messages

Reply:
My bad - not sure why/how i removed this one.